lib/enums.js

```diff
@@ -0,0 +1,11 @@
+const enums = exports;
+
+enums.swapProtocols = {
+  LND: 'LND',
+  RAIDEN: 'RAIDEN',
+};
+
+enums.orderSides = {
```

Comment:
We may be able to drop `orderSides` entirely if we use the convention of negative quantities meaning sells, positive meaning buys.